### DISCUSSION TITLE
ccf-validation-tools/Lymph_node/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#287] [General]
# ccf-validation-tools/Lymph_node/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Lymph_node/comments/

**Comment by emquardokus** [comment]
Lymph_Node_v1.3_DRAFT
https://docs.google.com/spreadsheets/d/15iuKGfnSjBbLd--873MKX2N0GgGKbXXk2dfzisiS-wg/edit#gid=1223566381

**Different labels**

1. FIXED typo    In row 21, the term CL:0009103 has different name/label in the source ontology. The name/label in the ASCT+B table is ymph node marginal reticular cell and the one in the ontology is lymph node marginal reticular cell. For reference, the given name/label by SMEs is Marginal Reticular Cell. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

2.  FIXED   In row 17, the term CL:0000669 has different name/label in the source ontology. The name/label in the ASCT+B table is pericyte cell and the one in the ontology is pericyte. For reference, the given name/label by SMEs is Pericyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

3. Corrected typo  lymph node germinal center dark zone Rows 37 and 38  In row 37, the term UBERON:8410053 has different name/label in the source ontology. The name/label in the ASCT+B table is ymph node germinal center dark zone and the one in the ontology is lymph node germinal center dark zone. For reference, the given name/label by SMEs is Dark Zone. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Blank ontology ID**
ADDED lymphatic capillary	UBERON:0006842 In row 59, no term id was found for the name/label capillaries of lymph node.
